html-css/trending/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Layout</title>
+    <style>
+        html,
+        body {
+            margin: 0;
+            background: rgb(242, 242, 242);
+            height: 99%;
+        }
+
+        .main-section {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            flex-wrap: wrap;
+            margin: auto;
+            width: 80%
+        }
+
+        .main-container {
+            margin: auto;
+        }
+
+        .topic {
+            margin-right: auto;
+            margin-bottom: 25px;
+            margin-left: 37px;
+            height: 120px;
+            border-radius: 5px;
+            background: white;
+            display: flex;
+            width: calc(50% - 75px);
+            flex-direction: row;
+        }
+
+        .topic-content {
+            padding: 40px;
+            width: 90%;
+            margin: auto;
+            font-family: Arial, Helvetica, sans-serif;
+            display: flex;
+            flex-direction: row;
+            align-content: stretch;
```

Comment:
I would add 
`
align-items: center;
justify-content: space-between;
`
This will center the `.arrow` and separate between the arrow and the text.

then `.arrow ` doesn't need margin definitions